src/robusta/core/reporting/finding_subjects.py

```diff
@@ -0,0 +1,52 @@
+from typing import Union, Optional
+from ...integrations.kubernetes.autogenerated.events import RobustaDeployment, RobustaJob, RobustaPod, v1ClusterRole, v1ClusterRoleBinding, v1DaemonSet, v1Event, v1HorizontalPodAutoscaler, v1Namespace, v1Node, v1ReplicaSet, v1Service, v1ServiceAccount, v1StatefulSet, v2beta1ClusterRole, v2beta1ClusterRoleBinding, v2beta1DaemonSet, v2beta1Event, v2beta1HorizontalPodAutoscaler, v2beta1Namespace, v2beta1Node, v2beta1ReplicaSet, v2beta1Service, v2beta1ServiceAccount, v2beta1StatefulSet, v2beta2ClusterRole, v2beta2ClusterRoleBinding, v2beta2DaemonSet, v2beta2Event, v2beta2HorizontalPodAutoscaler, v2beta2Namespace, v2beta2Node, v2beta2ReplicaSet, v2beta2Service, v2beta2ServiceAccount, v2beta2StatefulSet
+from .base import FindingSubject
+from .consts import FindingSubjectType
+
+
+class KubeObjFindingSubject(FindingSubject):
+    UnionKubernetesObjects = Optional[Union[
```

Comment:
The list of items here isn't constant.
On the events it's auto generated.
I don't think we get that much from this typed object.
I think it adds too much complexity for the benefits it provides